src/ui/src/templates/reset_interval_modal.html

```diff
@@ -0,0 +1,30 @@
+<div class="modal-content">
+       <div class="modal-header">
+              <h4>Reset the Job Interval?</h4>
+       </div>
+
+       <div class="modal-body">
+              This job has an interval trigger. Choose one of the buttons below to update when the <i>original job</i> should be run again:
+              <ul>
+                     <li>Selecting "Yes" updates the next run time of the original job based on the current time.</li>
+                     <li>Selecting "No" will keep the original job's existing next run time.</li>
+                     <li>Selecting "Cancel" cancels running the job now and no changes will be made to the next run time.</li>
+              </ul>
+       </div>
+
+       <div class="modal-footer">
+       <input type="button"
+              class="btn btn-primary"
+              value="Yes"
+              ng-click="setIntervalReset(true)" />
+       <input type="button"
+              class="btn btn-secondary"
+              value="No"
+              ng-click="setIntervalReset(false)" />
+       <input type="button"
+              class="btn btn-danger"
+              value="Cancel"
+              ng-click="cancelRunNow()" />
+       </div>
+</div>
+       
```

Comment:
The diff is claiming there's no newline at the end of this file. Is that an eslint rule you have?

Reply:
Ah, never mind it looks like that's what you added with the change